fast64_internal/mk64/__init__.py

```diff
@@ -0,0 +1,147 @@
+import math
+import bpy
+from bpy_types import PropertyGroup
+import mathutils
+from ..f3d.f3d_gbi import get_F3D_GBI
+from ..f3d.f3d_material import createF3DMat
+from ..f3d.f3d_parser import F3DContext, getImportData, importMeshC
+from ..panels import MK64_Panel
+from ..utility import prop_split
+from bpy.utils import register_class, unregister_class
+from ..utility import raisePluginError
+from .f3d_course_parser import processCommands, parseCourseVtx
+
+
+class MK64_Properties(PropertyGroup):
+    """Global MK64 Scene Properties found under scene.fast64.mk64"""
+
+    version: bpy.props.IntProperty(name="MK64_Properties Version", default=0)
+
+
+class MK64_ImportCourseDL(bpy.types.Operator):
+    # set bl_ properties
+    bl_idname = "object.f3d_course_import_dl"
```

Comment:
```suggestion
    bl_idname = "object.fast64_mk64_course_import_dl"
```
or something approaching (for uniqueness reasons)
this is probably widely inconsistent across the codebase but it's really not a problem